kloppy/domain/models/event.py

```diff
@@ -350,7 +350,9 @@ class PassType(Enum):
     THROUGH_BALL = "THROUGH_BALL"
     CHIPPED_PASS = "CHIPPED_PASS"
     FLICK_ON = "FLICK_ON"
+    SHOT_ASSIST = "SHOT_ASSIST"
     ASSIST = "ASSIST"
+    SHOT_ASSIST_2ND = "SHOT_ASSIST_2ND"
```

Comment:
What is the purpose of `SHOT_ASSIST_2ND`?

Reply:
That is a mistake. My original idea was to also add create pre-assist qualifiers, but think we should keep that out of scope of this PR. I'll fix it.

Reply:
I added it here as a sort of placeholder to make the data model more "complete", so that we have all possible options related to pre/normal and shot/goal assists: `SHOT_ASSIST` `ASSIST` `SHOT_ASSIST_2ND` `ASSIST_2ND`.

I don't think StatsPerform, StatsBomb or Wyscout has qualifiers for `SHOT_ASSIST_2ND`, but I can image this changing or there being other (future) providers that do add qualifiers for this.

But if you prefer to remove it from the data model as long as we don't have a data provider supporting it, happy to remove it.